File: src/main/java/com/eager/questioncloud/portone/PortonePayment.java
package com.eager.questioncloud.portone;

import lombok.Getter;

@Getter
public class PortonePayment {
    private String id;
    private PortonePaymentStatus status;
    private PortonePaymentAmount amount;
    private String receiptUrl;

    @Getter
    public static class PortonePaymentAmount {
        private int total;
        private int taxFree;
        private int vat;
    }
}

File: qc-external-pg-api/src/main/java/com/eager/questioncloud/pg/portone/PortonePayment.kt
package com.eager.questioncloud.pg.portone

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class PortonePayment(
    val id: String,
    val status: PortonePaymentStatus,
    val amount: PortonePaymentAmount,
    val receiptUrl: String,
)
